nucliadb_relations/src/storage_system/mod.rs

```diff
@@ -316,7 +272,7 @@ mod tests {
     fn open_and_create() {
         let dir = tempfile::tempdir().unwrap();
         StorageSystem::create(dir.path());
-        StorageSystem::open(dir.path());
+        StorageSystem::create(dir.path());
```

Comment:
You can remove this line :cat:

Reply:
Ok, sounds fair. Please add a comment to avoid to remove that line later on (because I am sure I will forget about it and feel tempted to remove it in a future iteration :cat:)

Reply:
Plus, can you remove the unnecessary `assert!(true)` just after ?